docs/kap_proposals/kap_7_remote_inventory.md

```diff
@@ -0,0 +1,64 @@
+# Remote Inventory Federation
+
+This feature would add the ability to Kapitan to fetch parts of the inventory from remote locations (https/git). This would allow users to combine different inventories from different sources and build modular infrastructure reusable across various repos.
+
+
+Author: @alpharoy14
+
+## Specification
+
+The configuration and declaration of remote inventories would be done in the hidden file `​.kapitan`.
```

Comment:
I think we should leave the `.kapitan` reserved for setting default kapitan flag values only

Reply:
So instead would a solution be to define a new `.kapitanmodules` file for example?

Reply:
I think using the inventory to add remotes would make more sense